src/spawn.ts

```diff
@@ -3,8 +3,8 @@ import { spawn } from "child_process";
 export default (callArguments: string[]): Promise<Buffer> => {
   return new Promise((resolve, reject) => {
     const process = spawn("qpdf", callArguments);
-    const stdout: string[] = [];
-    const stderr: string[] = [];
+    const stdout: Uint8Array[] = [];
```

Comment:
This one I am a little less sure about.  I had to console.log to figure out what the problem was.  It appears the the data we receive in the on listeners are Buffers (Uint8Array) and not strings.  So what was happening is that the Buffer.from was returning a malformed buffer as you can't BufferFrom and array of buffers.  You have to Buffer.concat them into a single buffer. 

This one is odd and I can't find any definitive docs that this changed at one point or what the actual type is supposed to be so a bit of guess and check here.